jetty-core/jetty-compression/jetty-compression-brotli/src/main/java/org/eclipse/jetty/compression/brotli/BrotliCompression.java

```diff
@@ -159,7 +161,7 @@ public InputStream newDecoderInputStream(InputStream in, DecoderConfig config) t
     public DecoderSource newDecoderSource(Content.Source source, DecoderConfig config)
     {
         BrotliDecoderConfig brotliDecoderConfig = BrotliDecoderConfig.class.cast(config);
```

Comment:
Don't use `cast()` in this way. It is enough a normal cast:

```suggestion
        BrotliDecoderConfig brotliDecoderConfig = (BrotliDecoderConfig)config;
```